packages/server/src/utilities/rowProcessor/tests/outputProcessing.spec.ts

```diff
@@ -177,10 +177,6 @@ describe("rowProcessor - outputProcessing", () => {
       expect(output2.attach[0].url?.split("?")[0]).toBe(
         "/files/signed/prod-budi-app-assets/test.jpg"
       )
-
-      row.attach[0].url = "aaaa"
```

Comment:
I think the right test here would be having the `url` but no `key`. If this ever happen (I don't really see how) we want that to be the behaviour